app/templates/partials/summary/summary.html

```diff
@@ -4,10 +4,9 @@
   <div class="summary">
 
     {% for group in content.summary.groups %}
-
-      {% if group.title %}
-      <h2 class="summary__title saturn" id="{{group.id}}">{{group.title}}</h2>
-      {% endif %}
+      {% if group.title and content.summary.summary_type == 'Summary' %}
+        <h2 class="summary__title saturn" id="{{group.id}}">{{group.title}}</h2>
+        {% endif %}
```

Comment:
I think we should remove the additional logic around summary type here to ensure consistency across the summary pages.